wiki/BanchoBot/id.md

```diff
@@ -1,25 +1,142 @@
-
 *Untuk daftar perintah yang menggunakan slash (klien), kunjungi artikel [Konsol Obrolan](/wiki/Chat_Console#daftar-perintah).*
 
-| Perintah | Penjelas | Deskripsi | Contoh Masukan | Contoh Tanggapan |
-| :-- | :-- | :-- | :-- | :-- |
-| `HELP` | Tidak ada | Menampilkan daftar perintah yang tersedia | `!help` | This list |
-| `ROLL` | `[number]` | Secara acak akan mendapatkan nilai hingga ada nomor yang terpilih (standar adalah 100). | `!roll 1000` | "Nathanael rolls 789 point(s)" |
-| `STATS` | `<user_name>` | Menampilkan statistik pengguna bergantung pada mode yang mereka mainkan | `!stats peppy` | "Stats for [peppy](https://osu.ppy.sh/users/2): Score: 412,018,739 (#94718), Plays: 7073 (lv65), Accuracy: 87.31%" |
-| `WHERE` | `<user_name>` | Menampilkan lokasi pengguna saat ini. | `!where Kyubey` | "Kyubey is in Russia" |
-| `FAQ` | `[lang:]<entry>/list` | Menampilkan pesan entri. Gunakan `list` sebagai entri untuk menampilkan semua entri yang tersedia. | `!faq ping` | "Pong!" |
-| `REPORT` | `<user_name> <reason>` | Mengirimkan laporan ke moderator chat. Ganti spasi dalam nama pengguna dengan garis bawah `_` | `!report S_o_h spamming in #japanese` | "Chat moderators have been alerted. Thanks for your help." |
-| `REQUEST` | Tidak ada | Menerima beatmap dari pemain yang meminta bantuan modding secara acak. | `!request` | "[HoneyWorks - Tokyo Summer Session feat. CHiCO](https://osu.ppy.sh/beatmapsets/426252#osu) by MrSergio" |
+Untuk menggunakan perintah BanchoBot, kamu harus menggunakan awalan `!` setiap saat akan memulai perintah dan setelahnya diikuti dengan nama perintah menggunakan awal kapital (tanpa spasi didalamnya). Perintah ini dapat digunakan di dalam kanal obrolan dan obrolan privat dengan BanchotBot.
+
+Jika pengguna mencoba mengirim perintah dalam obrolan publik, pengguna biasa tidak dapat melihatnya dan pemain yang mengirim perintah akan mendapat jawabannya sebagai pesan pribadi. Pengguna juga dapat menggunakan perintah `/bb <command>` untuk membuka tab BanchoBot secara otomatis dan mengirim perintah dengan segera.
+
+Daftar perintah BanchoBot:
+
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#stats)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
+
+### Help
+
+```
+!help
+```
+
+`!help` menampilkan seluruh perintah dari BanchoBot. Berikut contoh hasil dari perintah tersebut:
+
+```
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
+```
+
+*Perhatian: Perintah `!request` tidak lagi didukung oleh BanchoBot.*
+
+### Roll
+
+```
+!roll <argument>/<number>
```

Comment:
i think you can actually translate the arguments in angle brackets for all commands, as it describes the command and is not a part of any BanchoBot response

Reply:
done